packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.html

```diff
@@ -31,16 +31,23 @@
             </template>
 
             <template if:false={isFacetSearchActive}>
-              <template for:each={values} for:item="v">
-                <c-quantic-facet-value 
-                  onselectvalue={onSelectValue} 
-                  key={v.value} 
-                  item={v} 
-                  is-checked={v.checked}
-                  data-cy={v.value}
-                  display-as-link={isDisplayAsLink}>
-                </c-quantic-facet-value>
-              </template>
+              <fieldset> 
+                <legend class="slds-assistive-text">Test</legend> 
```

Comment:
Why did you put `Test` inside the `legend` element? Shouldn't it contain the facet title or something like that instead?